src/database.py

```diff
@@ -0,0 +1,171 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine, Column, Integer, String, Float, DateTime, Sequence
+from sqlalchemy.ext.declarative import declarative_base
+from contextlib import contextmanager
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+db_path = "customer_orders.duckdb"
+Base = declarative_base()
+
+@contextmanager
+def get_db():
+    sql_alchemy_database_url = f"duckdb:///{db_path}"
+    engine = create_engine(sql_alchemy_database_url)
+    Base.metadata.create_all(engine)
+    SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=create_engine(sql_alchemy_database_url))
+    db = SessionLocal()
+    try:
+        logger.info("Database session opened")
+        yield db
+    except Exception as e:
+        logger.error(f"Error occurred during database operation: {str(e)}")
+        raise
+    finally:
+        logger.info("Closing database session")
+        db.close()
+
+
+class Sellers(Base):
+    __tablename__ = "sellers"
+
+    id = Column(Integer, Sequence('id'), primary_key=True)
+    seller_id = Column(String)
+    seller_zip_code_prefix = Column(Integer)
+    seller_city = Column(String)
+    seller_state = Column(String)
+
+class Customers(Base):
```

Comment:
I see no max_length constraints on any field for your models, I think some fields would need this here